docs/build-on-btc/index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+keywords: [bitcoin, btc, chain fusion, bitcoin integration, btc integration, build on bitcoin, how to build btc, btc dapps, building btc dapps, build bitcoin, bitcoin dapps, build btc dapps, build btc apps]
+hide_table_of_contents: true
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Build on Bitcoin
+
+<MarkdownChipRow labels={["Bitcoin" ]} />
+
+ICP brings smart contract functionality to Bitcoin and opens up new possibilities for dapp development.
+
+Through a protocol-level integration with the Bitcoin network, smart contracts deployed on ICP can interact with the Bitcoin network directly without using a bridge or oracle.
+
+Threshold signing APIs, including [threshold ECDSA](/docs/building-apps/network-features/signatures/t-ecdsa) and [threshold Schnorr](/docs/building-apps/network-features/signatures/t-schnorr), enable an ICP smart contract to hold assets like BTC or [sign](/docs/build-on-btc/btc-transactions/sign-transactions) and [submit transactions](/docs/build-on-btc/btc-transactions/submit-transactions) to the Bitcoin network.
+
+When an ICP smart contract wants to submit a Bitcoin transaction, the following steps must occur:
```

Comment:
Nit: "must" is a bit too strong. For example, it is possible that a dapp already knows its Bitcoin address(es), maybe it even caches its UTXOs.
How about weaken the statement a little bit (e.g., "typically occur" or "may occur")?